components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -111,15 +113,29 @@ - (instancetype)initWithFrame:(CGRect)frame {
     _displayMaskLayer.fillColor = [UIColor whiteColor].CGColor;
 
     _titleLabel = [[UILabel alloc] initWithFrame:CGRectZero];
-    _titleLabel.font = [UIFont mdc_standardFontForMaterialTextStyle:MDCFontTextStyleTitle];
+    // If we are using the default (system) font loader, retrieve the
+    // font from the UIFont standardFont API.
+    if ([MDCTypography.fontLoader isKindOfClass:[MDCSystemFontLoader class]]) {
```

Comment:
This is not correct anymore since we are going to add a fontloader for each style. Or we are creating a delegate. This is how we globally set a default font faces for our styles.

The MDCSystemFontLoader should already return the [UIFont mdc_standardFontForMaterialTextStyle:kTitleTextStyle]

concretely we should drop the if clause and just put:
`_titleLabel.font = [MDCTypography titleFont];`

Reply:
by copying this logic that will be in [MDCTypography titleFont] you are making it harder for us to pivot if we determine that UIAppearance is not scalable for our needs in GMDC

Reply:
by accessing MDCTypography.fontloader you are increasing usage of the very thing you are trying to delete.

MDCTypography

Reply:
Do we have evidence the UIAppearance is not scalable?
We already evaluated UIAppearance and decided it was the path forward when we added the color themers.  Happy to revisit that decision, but it might be out of scope for this PR.

As far as utility:

If I want all of my buttons to use GoogleSans:
[MDCButton appearance].titleFont = [UIFont fontNamed:@"Google-Sans"];

Following the pattern of our color themers if we want to turn use Roboto everywhere
MDCFontScheme *scheme = [MDCFontScheme schemeWithFontFamily:@"Roboto"];
[MDCAppBarFontThemer apply:scheme to: [MDCAppBarView appearance]];
[MDCButtonFontThemer apply:scheme to:[MDCButton appearance]];
[MDCFeatureHighlightThemer apply:scheme to:[MDCFeatureHighlightThemer appearance]];
// etc.